src/main/java/net/neoforged/camelot/log/JoinsLogging.java

```diff
@@ -33,7 +33,7 @@ public void onEvent(@NotNull GenericEvent gevent) {
                     .setTitle("User Joined")
                     .addField("User", event.getMember().getUser().getName() + " (" + event.getMember().getAsMention() + ")", true)
                     .addField("Account age", "**" + Duration.ofSeconds(OffsetDateTime.now().toEpochSecond() - event.getUser().getTimeCreated().toEpochSecond()).toDays() + " days**", true)
-                    .addField("Member count", String.valueOf(event.getGuild().getMemberCount()), true)
+                    .addField("Joined Discord", TimeFormat.RELATIVE.format(event.getUser().getTimeCreated()), true)
```

Comment:
Why did you remove the member count? Also, the account age is when the user joined discord but in days.